chia/consensus/blockchain.py

```diff
@@ -720,8 +714,6 @@ async def validate_unfinished_block(
             uint32(prev_height + 1),
             None if npc_result is None else npc_result.conds,
             fork_info,
-            None,
-            validate_signature=False,  # Signature was already validated before calling this method, no need to validate
```

Comment:
the signatures have already been validated, and recorded in the `npc_results.conds`